libkbfs/md_ops.go

```diff
@@ -563,6 +563,13 @@ func (md *MDOpsStandard) PruneBranch(
 	return md.config.MDServer().PruneBranch(ctx, id, bid)
 }
 
+// ResolveBranch implements the MDOps interface for MDOpsStandard.
+func (md *MDOpsStandard) ResolveBranch(
```

Comment:
Although it looks like `journalMDOps` can still return an error if there's no journal for a particular `TlfID`. Hmm. So maybe we should have a standard error that we return when the TLF doesn't support `ResolveBranch`, and have `MDOpsStandard` always return it? I'm not sure what's the best way, since I don't know what the callers look like.